.circleci/config.yml

```diff
@@ -57,45 +60,42 @@ jobs:
       - run:
           name: Verify binary compatibility with supported IntelliJ versions
           command: ./scripts/verify-binary-compatibility
-
       - run:
-          # TODO (#274): use artifacts published on Bintray/JCenter instead
-          # Note: this step (at least building&publishing the ide-probe artifacts) can't be baked into the Docker image
-          # since the VirtuslabRnD/ide-probe is a private repository as for now,
-          # and also we can't publish the binary artifacts due to VL's client's policies.
+          # To be removed once Docker CI image is regenerated
           name: Prepare prerequisites for UI tests
           command: |
-            set -x
-            apt-get update
-            apt-get install -y gnupg
-            echo "deb https://dl.bintray.com/sbt/debian /" | tee -a /etc/apt/sources.list.d/sbt.list
-            # keyserver.ubuntu.com is exceptionally flaky, let's keep the public key in the repo instead of downloading
-            # from https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823
-            apt-key add < sbt-debian.pub
-            apt-get update
-            apt-get install -y sbt
-            git clone https://${GITHUB_TOKEN}@github.com/VirtuslabRnD/ide-probe.git ~/ide-probe
-            cd ~/ide-probe/
-            git checkout 674acde5e17d9479733dd55faf880ea162248e37  # master as of 2020-05-14
-            # Actually, all the commands up to this point take ~30 seconds;
-            # it's sbt invocation itself that's the most time-consuming.
-            sbt '; api/publishM2; probePlugin/publishM2; driver/publishM2; junitDriver/publishM2'
-      - run:
-          name: Compile UI tests
-          command: ./gradlew -PenableUiTests :uiTests:compileTestScala
-      - run:
-          name: Run UI tests
-          command: |
-            # To be removed once Docker CI image is regenerated
+            command -v nc || {
+              apt-get update && apt-get install -y netcat
+            }
+
             command -v xauth || {
               apt-get update && apt-get install -y xauth
             }
 
-            # stop, max-workers, jvmargs: limit Gradle's memory consumption so that enough space is left for IntelliJ
-            # enableUiTests: see uiTests/build.gradle
-            # debug: let's leave it for some time until we're sure that these tests are reliable
-            ./gradlew --stop
-            ./gradlew -PenableUiTests --debug --max-workers=1 -Dorg.gradle.jvmargs=-Xmx256M :uiTests:test
+            dir="/root/.java/.userPrefs/jetbrains/_!(!!cg\"p!(}!}@\"j!(k!|w\"w!'8!b!\"p!':!e@=="
```

Comment:
See Dockerfile for the meaning  of what's happens here, this step is temporary (should be baked into Docker image)